Source/WebKit/UIProcess/API/ios/WKWebViewIOS.h

```diff
@@ -156,6 +156,8 @@ enum class TapHandlingResult : uint8_t;
 - (void)_beginLiveResize;
 - (void)_endLiveResize;
 
++ (void)_resetPresentLockdownModeMessage;
```

Comment:
This could be moved to `WKWebViewPrivateForTestingIOS.h`.

Reply:
Sounds good. It is actually backed by a static 'bool' in the `WKWebViewIOS.mm` file, but I can expose it through the `WKWebViewPrivateForTestingIOS.h `interface.